pyproject.toml

```diff
@@ -1,5 +1,73 @@
 [build-system]
-requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4", "wheel", "numpy"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-vcs"]
```

Comment:
Would it be good idea to pin versions here so new hatch release won't accidentally break the build?

Reply:
Probably! I'll do that shortly.